optuna/storages/_journal/storage.py

```diff
@@ -135,11 +134,15 @@ def _sync_with_backend(self) -> None:
         logs = self._backend.read_logs(self._replay_result.log_number_read)
         self._replay_result.apply_logs(logs)
 
-    def create_new_study(self, study_name: Optional[str] = None) -> int:
+    def create_new_study(
+        self, directions: Sequence[StudyDirection], study_name: Optional[str] = None
+    ) -> int:
         study_name = study_name or DEFAULT_STUDY_NAME_PREFIX + str(uuid.uuid4())
 
         with self._thread_lock:
-            self._write_log(JournalOperation.CREATE_STUDY, {"study_name": study_name})
+            self._write_log(
+                JournalOperation.CREATE_STUDY, {"study_name": study_name, "directions": directions}
```

Comment:
Here you added `directions` attribute in the `CREATE_STUDY` log, so that you must also update the implementation of `_apply_create_new_study`.